sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -150,6 +150,60 @@ internal static LusidInstrument CreateExampleInterestRateSwap()
                 instrumentType: LusidInstrument.InstrumentTypeEnum.InterestRateSwap
             );
         }
+        
+        internal static InterestRateSwap CreateSwapByNamedConventions(DateTimeOffset startDate, DateTimeOffset maturityDate, decimal fixedRate, FlowConventionName flowConventionName, FlowConventionName indexConventionName, string fixedLegDirection = "Pay", decimal notional=100m)
```

Comment:
All of the other functions in this file have self explanatory names. This one is a bit less obvious, can you add a docstring to this one?